static/app/actionCreators/monitors.tsx

```diff
@@ -66,3 +66,29 @@ export async function updateMonitor(
 
   return null;
 }
+
+export async function setEnvironmentIsMuted(
+  api: Client,
+  orgId: string,
+  monitorSlug: string,
+  environment: string,
+  isMuted: boolean
+) {
+  addLoadingMessage();
+
+  try {
+    const resp = await api.requestPromise(
+      `/organizations/${orgId}/monitors/${monitorSlug}/environments/${environment}`,
+      {method: 'PUT', data: {isMuted}}
```

Comment:
does this send a `key/value` of `isMuted: true/false`?

Reply:
yup!